cylc/flow/task_pool.py

```diff
@@ -1629,23 +1629,21 @@ def match_future_tasks(
                 LOG.warning(f"No active tasks matching: {id_}")
                 unmatched_tasks.append(id_)
                 continue
+            if not tokens['task']:
+                # make task globs explicit to make warnings clearer
+                tokens['task'] = '*'
```

Comment:
I think if this is moved to above the previous `if` condition, then the `No active tasks in the family` message below would be fine

Reply:
The "in family" bit is misnomer, for the reported example the task is unspecified i.e. "*", so reporting it this way is confusing. Clearer to say "no tasks match ...".

Reply:
But if this is moved above this bit:
https://github.com/cylc/cylc-flow/blob/1f6364d1a02187850e1da5a9c9c222f5992e066f/cylc/flow/task_pool.py#L1623-L1628

then the only way of reaching here:
https://github.com/cylc/cylc-flow/blob/1f6364d1a02187850e1da5a9c9c222f5992e066f/cylc/flow/task_pool.py#L1637-L1639

is as the comment says, if the name is a family name. So it wouldn't be a misnomer